go/vt/vttablet/tabletmanager/action_agent.go

```diff
@@ -252,6 +254,11 @@ func NewActionAgent(
 		return nil, err
 	}
 
+	tabletType, err := validateDemoteMasterType(*demoteMasterType)
```

Comment:
The var name `tabletType` might mislead a reader to think it's the type that will be used at startup (since this is inside NewActionAgent). Let's call it something like `demoteMasterTabletType`.

Reply:
makes sense, done!